src/styles/index.css

```diff
@@ -39,10 +39,33 @@
 body {
   font-family: 'PilcrowRounded';
   font-size: 16px;
+  color: #2e2e2e;
 
-  @apply bg-gray-50
+  @apply bg-gray-50;
 }
 
-.multiSelectContainer li:hover{
+.multiSelectContainer li:hover {
   background: #f28316 !important;
-}
\ No newline at end of file
+}
+
+.underline {
```

Comment:
there's class called `underline` in tailwind already, no need to recreate it here